src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -855,6 +855,11 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                             ->end()
                         ->end()
                         ->arrayNode('auto_mapping')
+                            ->info('A collection of namespaces for which auto-mapping will be enabled.')
```

Comment:
That's describing the keys - should we describe the values also?

Reply:
I [changed this](https://github.com/symfony/symfony/compare/13461edaaaf880c12420323958c0b86864d5e80f..8eb29a7b73c2d4e410a7fcdaef199326d1308fbe) after @weaverryan 's suggestion in https://github.com/symfony/symfony/pull/34707#discussion_r355455646 actually. Perhaps we can find something between, or just add info on the `services` node below.